pyproject.toml

```diff
@@ -23,7 +23,7 @@ packages = [{ include = "django_twined" },]
 python = ">=3.9,<3.11"
 django = ">=3.0,<4.0"
 channels = ">=3.0,<4.0"
-octue = "^0.39.0"
+octue = "^0.46.3"
```

Comment:
Maintaining backward compatibility is definitely an area where I think you could improve. There's no new feature in the octue library that enables what's added here, so there's no reason to tighten the version range.

What this tighter constraint means is that if I'm running an app and want to add this one little tweak in django twined, I have to wade through several breaking changes in the octue library wherever it's used across the whole of the rest of my app.

Reply:
Yes, it is a good *opportunity* for the developer-users to upgrade a third library at the same time. But this constraint turns it from an opportunity into being forced. Which just makes life really hard for people!

Reply:
(it's the responsibility of the owners of that app to decide when to upgrade the third party library, not yours).

Think about all the problems we have with the google cloud SDK compatibilities - they often arise because those libraries decide to force you into tight version ranges in adjacent libraries.